include/aspect/material_model/rheology/constant.h

```diff
@@ -0,0 +1,64 @@
+/*
+  Copyright (C) 2019 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef _aspect_material_model_rheology_constant_h
+#define _aspect_material_model_rheology_constant_h
+
+#include <aspect/global.h>
+
+namespace aspect
+{
+  namespace MaterialModel
+  {
+    using namespace dealii;
+
+    namespace Rheology
```

Comment:
RheologyModel?

Reply:
Personally I want to get rid of as many `model` as possible (we renamed all the boundary conditions and initial conditions for 2.0). I can live with `MaterialModel`, but I do not want to introduce more of them :smile:.